### DISCUSSION TITLE
Can I use this with a VM instead of Docker?

### DISCUSSION THREAD

**Discussion by ballo** [wg-easy/wg-easy#1220] [General]
As much as I could like to use Docker, it doesn't support USB passthrough on MacOS so I have to use a VM like Parallels. 
(In fact, the only reason I'm trying to get wireguard working is to work around a bug where I can't use the VM as the gateway for the Mac host)
Is it possible to use wg-easy to setup my VPN or is it Docker specific?

**Comment by tetuaoro** [comment]
Hi,

Sure you can. You need 3 things inside your VM :
- clone this repo
- wireguard-tools installed
- and package manager like npm.

To start the vpn, just run `node server.js` in `src` directory. Don't forget to open and bind your ports.

Check this following [link](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker).

**Comment by ballo** [comment]
I managed to get it working manually (sorta) but I'll keep this in mind. Thanks. 

FYI the home page says Docker is required.
